pilot/pkg/config/kube/crdclient/client.go

```diff
@@ -163,6 +164,7 @@ func NewForSchemas(client kube.Client, revision, domainSuffix string, schemas co
 			GroupVersionResource()).Informer(),
 		beginSync:   atomic.NewBool(false),
 		initialSync: atomic.NewBool(false),
+		scope:       log.WithLabels("controller", identifier),
```

Comment:
```suggestion
		scope:       scope.WithLabels("controller", identifier),
```

otherwise we are in default scope instead of "kube"